addons/website_event_track/views/event_track_templates_page.xml

```diff
@@ -203,7 +203,11 @@
                 class="tab-content collapse d-md-block o_wesession_track_aside_tabs">
                 <div class="tab-pane fade show active" id="track_list" role="tabpanel">
                     <ul class="list-group list-group-flush">
-                        <li t-foreach="tracks_other" t-as="track_other" class="list-group-item list-group-item-action">
+                        <t t-set="has_ongoing_or_upcoming"
+                           t-value="any(track.is_track_live or track.track_start_remaining for track in tracks_other)"/>
+                        <li t-foreach="tracks_other" t-as="track_other"
+                            t-if="not (has_ongoing_or_upcoming and track_other.is_track_done)"
+                            class="list-group-item list-group-item-action">
```

Comment:
Do we still need those changes if we remove all the tracks we don't want to show in the parent template `event_track_main`?

Reply:
yes, we don't need it Thanks for pointing out :+1: